tap_shiphero/sync.py

```diff
@@ -61,7 +49,19 @@ def sync_products(client, catalog, state, start_date, end_date, stream_id, strea
 
     write_schema(catalog, stream_id)
 
-    last_date = get_bookmark(state, stream_id, start_date)
+    last_date = bookmarks.get_bookmark(state, stream_id, 'datetime', start_date)
+
+    # Rip this out once all bookmarks are converted
+    old_style_bookmark = bookmarks.get_bookmark(state, stream_id, stream_id)
+    if old_style_bookmark:
```

Comment:
Should be checking if `isinstance(old_style_bookmark, str)` or `old_style_bookmark is not None`.